apps/web/pages/team/[slug]/[type]/embed.tsx

```diff
@@ -1,104 +1,7 @@
-import type { GetServerSidePropsContext } from "next";
-import { z } from "zod";
+import withEmbedSsr from "@lib/withEmbedSsr";
```

Comment:
It was a copy paste of team/[slug]/[type], so importing that only here like it's done for other embed routes.